Python/ceval.c

```diff
@@ -1571,10 +1597,7 @@ typedef struct {
     PyObject *kwnames;
 } CallShape;
 
-static inline bool
-is_method(PyObject **stack_pointer, int args) {
-    return PEEK(args+2) != NULL;
-}
+#define is_method(stack_pointer, args) (PEEK((args)+2) != NULL)
```

Comment:
If it must be a macro to avoid reintroducing gh-89279, maybe add a comment to explain that?